examples/README.md

```diff
@@ -0,0 +1,135 @@
+## Snap Plugin Go Library Examples
+Here you will find example plugins that cover the basics for writing collector, processor, and publisher plugins.
+
+## Build Plugins & Use with Snap
+
+To get these example collector, processor, and publisher plugins to build properly and work with Snap you will need to have [glide](https://glide.sh/) installed on your PATH. You should also add snapctl and snapd to your PATH. 
+
+To test these plugins with Snap, you will need to have [Snap](https://github.com/intelsdi-x/snap) installed, check out these docs for [Snap setup details](https://github.com/intelsdi-x/snap/blob/master/docs/BUILD_AND_TEST.md#getting-started).
+
+###1. Get the plugin library repo:
+`go get github.com/intelsdi-x/snap-plugin-lib-go` will add the repo to your $GOPATH
```

Comment:
you can use either `go get github.com/intelsdi-x/snap-plugin-lib-go/...` or `go get github.com/intelsdi-x/snap-plugin-lib-go/v1/plugin`.

The go get command actually works with packages not repos. The error that shows up is because there is no top-level package. I think it could also be addressed by having an empty `main.go` at the top level.